CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptPkcs7VerifyCommon.c

```diff
@@ -470,6 +470,10 @@ Pkcs7GetSignedData (
     goto End;
   }
 
+  if (CertNum != 0) {

```

Comment:
When parsing the certificate fails or verify ends, the cert pools are not released correctly. 
This problem requires a more robust fix.